app/javascript/lib/validation/validate-file-content.js

```diff
@@ -492,10 +599,31 @@ function validateClusterCoordinates(headers) {
   return issues
 }
 
+/** parse a metadata file, and return an array of issues, along with file parsing info */
+export async function parseBarcodesFile(chunker, mimeType, fileOptions) {
+  let issues = []
+
+  const dataObj = {} // object to track multi-line validation concerns
+  await chunker.iterateLines((rawline, lineNum, isLastLine) => {
+    issues = issues.concat(validateUniqueRowValuesWithinFile(rawline, isLastLine, dataObj))
+  })
+  return { issues }
+}
+
+/** parse a metadata file, and return an array of issues, along with file parsing info */
+export async function parseFeaturesFile(chunker, mimeType, fileOptions) {
```

Comment:
nice, I like how you made this a separate function, even though, for now, it's a a duplicate of parseBarcodesFile